api_tests/e2e/rfc003/btc_eth/resume.ts

```diff
@@ -109,4 +109,4 @@ setTimeout(function() {
     );
 
     run();
-}, 60000);
+}, 120_000);
```

Comment:
NO!

This has to be 0 :see_no_evil: 

This has nothing to do with the timeout of the test but is a _delay_ up until the tests will actually run. We only need this because we run mocha with a special flag where we need to call `run` at the end. This allows us to do async work before running the tests.

Sorry about this, you cannot possible know this :sweat_smile: 

This can be removed once we migrated all of our tests.

Reply:
All good, glad you caught it :)